packages/opentelemetry-web/src/WebTracerFactory.ts

```diff
@@ -0,0 +1,35 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { WebTracer } from './WebTracer';
+import {
+  BasicTracer,
+  AbstractBasicTracerFactory,
+  BasicTracerConfig,
+} from '@opentelemetry/tracing';
+
+export class WebTracerFactory extends AbstractBasicTracerFactory {
```

Comment:
Same here, I think it should just inherit from a more general `AbstractTracerFactory`